src/services/error-manager.js

```diff
@@ -0,0 +1,15 @@
+const ApiErrorDeserializer = require('../deserializers/api-error');
+
+function handleError(rawError) {
+  let error;
+  // NOTICE: We check if the errors are from the API or if they've been thrown internally.
+  if (!rawError.status) {
+    error = rawError;
+  } else {
+    error = ApiErrorDeserializer.deserialize(rawError);
+  }
+
+  return error.message ? error.message : 'Oops something went wrong';
+}
+
+module.exports = handleError;
```

Comment:
[MISC] I have kind of an issue with this: this fie is called `error-manager` but directly exports a function.

Indeed it does not look like a service but more of a utility function file.

What about moving this file to `src/utils`, call it just `error.js` and expose the function as suggested:

```suggestion
module.exports = { handleError };
```

This way you can import it like this:

```
const { handleError } = require('../utils/error');
```